testing/irtest/cmd.cpp

```diff
@@ -69,20 +69,6 @@ static void print_cmd(const cmd *cmd)
   printf(")\n");
 }
 
-static int print_cmd(const char *name)
```

Comment:
called at line 76?

Reply:
it's another print_cmd at line 61, not this one.

Reply:
Ok, it's overloaded.

Reply:
I am curious why CI can't catch this warning.